tests/projects/local-kit/kits/extra-1-kit/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "extra-1-kit"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[package.metadata.build-kit]
+included-packages = [
+    "pkg-b",
+    "pkg-d",
+]
+
+[lib]
+path = "../kit.rs"
+
+[build-dependencies]
+pkg-b = { path = "../../packages/pkg-b" }
+pkg-d = { path = "../../packages/pkg-d" }
```

Comment:
Might be worth an `extra-3-kit` that includes `pkg-e` which:
* depends on `extra-1-kit` and `extra-2-kit` in its Cargo.toml
* BuildRequires `pkg-a`, `pkg-b`, `pkg-c`, `pkg -d` in its spec

That will test the implicit kit dependency we actually care about: `pkg-e` should have access to `core-kit` as an indirect dependency of `extra-1-kit` and `extra-2-kit`.

Reply:
Good idea to add this.